### DISCUSSION TITLE
Is it possible to use older HLS version?

### DISCUSSION THREAD

**Discussion by tsaiyuen** [bluenviron/mediamtx#1747] [Questions]
Hello! I need to stream some videos, but I have a HLS version limitation.. I saw that we are streaming in HLS v9 (`#EXT-X-VERSION:9`).
Is it possible to force it to HLS V3? Thank You!

**Comment by aler9** [comment]
Hello, set `hlsVariant` to `mpegts` as documented in the configuration file:

```yml
# Variant of the HLS protocol to use. Available options are:
# * mpegts - uses MPEG-TS segments, for maximum compatibility.
# * fmp4 - uses fragmented MP4 segments, more efficient.
# * lowLatency - uses Low-Latency HLS.
hlsVariant: mpegts
```

**Reply by tsaiyuen** [reply]
Thanks!!!!

**Reply by tsaiyuen** [reply]
This way we change all the hls from all transmissions, correct? Is there any way to change it for one specific channel or via query string?

**Reply by aler9** [reply]
HLS streams are generated once and served to all users in the same way, therefore the parameter is global. Nonetheless, there's a feature request to set `hlsVariant` on a path basis: #967

[a comment by an automated account is not shown]
